projects/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+    def __str__(self):
```

Comment:
Line 7에 한 줄 띄워주세요! 컨벤션은 기본적으로 맞춰주세요~

```suggestion

    def __str__(self):
```

Reply:
넵